stylesheets/components/image.css

```diff
@@ -0,0 +1,7 @@
+.img-container {
+border:2px solid var(--charcoal-grey);
+padding:2rem;
+}
+.responsive-img {
+  width: 100%;
```

Comment:
If you want an image to scale down if it has to, but never scale up to be larger than its original size, use 
max-width: 100%

you also read about object-fit: cover; that will help you for responsive image

Reply:
didn't know about `object-fit: cover`, thanks will look into this property